src/test/java/com/thindeck/dynamo/ClasspathRule.java

```diff
@@ -94,6 +94,7 @@ public boolean apply(final Class<?> input) {
                         && !name.endsWith("ITCase")
                         && !name.endsWith("ClasspathRule")
                         && !name.endsWith("RepoRule")
+                        && !name.equals(RegionLocalDynamo.class.getName())
```

Comment:
why did you add this condition?
to pass ```ImmutabilityTests``` you need to make ```RegionLocalDynamo``` immutable.
In your case it's pretty simple - just add  ```@Immutable``` to that class